conda_build/skeletons/cpan.py

```diff
@@ -28,6 +27,7 @@
 from conda_build.config import get_or_merge_config
 from conda_build.utils import on_win, check_call_env
 from conda_build.variants import get_default_variant
+from conda_build.version import _parse as parse_version
```

Comment:
I think we need to do this _everywhere_ where `LegacyVersion` could have been expected. So essentially we need to replicate this change in every other instance of `from pkg_resources import parse_version`.

Reply:
For the PyPI skeleton, I don't think that's needed given that PyPI will provide PEP 440 compatible versions, but the variants handling you're right.

Reply:
Interesting how do they handle the long tail of packages that preceded PEP 440?